src/rules/no-export-load-in-svelte-module-in-kit-pages.ts

```diff
@@ -0,0 +1,54 @@
+import type * as ESTree from "estree"
+import { createRule } from "../utils"
+import { isKitPageComponent, hasSvelteKit } from "./kit-helpers/kit-helpers"
+
+export default createRule("no-export-load-in-svelte-module-in-kit-pages", {
+  meta: {
+    docs: {
+      description:
+        "Disallow exporting load functions in `*.svelte` module in Svelte Kit page components.",
+      category: "Possible Errors",
+      recommended: true,
```

Comment:
Since changes to shareable configs are marked as breaking changes, could you please set recommended to false once?

```suggestion
      // TODO Switch to recommended in the major version.
      // recommended: true,
      recommended: false,
```

Reply:
updated. https://github.com/ota-meshi/eslint-plugin-svelte/pull/281/files#diff-086306a09a0074b23aac04f4b43be642ab718b55021155d2d4eccd9b233aaf59R11-R12